content/sensu-enterprise/3.0/upgrading.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Upgrading Sensu Enterprise"
+weight: 14
+product: "Sensu Enterprise"
+version: "3.0"
+menu: "sensu-enterprise-3.0"
+---
+
+Upgrading Sensu Enterprise is usually a straightforward process. In most cases,
+upgrading Sensu Enterprise only requires upgrading to the
+latest package. Certain versions of Sensu Enterprise may include changes that
+are *not backwards compatible* and require additional steps be taken when
+upgrading.
+
+- [Upgrading from Sensu Enterprise < 3.0](#upgrading-from-sensu-enterprise-3-0)
+	- [Changes in OpsGenie integration](#changes-in-opsgenie-integration)
+		- [Configure OpsGenie for `responders`](#configure-opsgenie-for-responders)
+		- [Update `overwrites_quiet_hours`](#update-overwrites-quiet-hours)
+	- [Changes in Java package dependency](#changes-in-java-package-dependency)
+- [Upgrading the Sensu Enterprise package](#upgrading-the-sensu-enterprise-package)
+
+## Upgrading from Sensu Enterprise < 3.0
+
+The following documentation provides steps necessary when upgrading
+from a Sensu Enterprise version prior to 3.0.
+
+### Changes in OpsGenie integration
+
+OpsGenie has deprecated and will shut down their v1 API on
+June 30, 2018.
+
+_NOTE: To continue using the Sensu Enterprise OpsGenie integration, you must
+upgrade to Sensu Enterprise 3.0 before June 30, 2018._
```

Comment:
Forgot to include this is my review ><, but what do you think about changing this to a `WARNING` callout?

Reply:
@apaskulin I think that makes sense. Would you say the `NOTE` in the next section should also be a warning?

Reply:
That's a good point. It might be better to consolidate into a single `WARNING`:

```
_WARNING: To continue using the Sensu Enterprise OpsGenie integration, you must
upgrade to Sensu Enterprise 3.0 and update your Sensu Enterprise OpsGenie configuration before June 30, 2018._
```